src/components/FullTags/TagsCombinaisonsForm.vue

```diff
@@ -0,0 +1,378 @@
+<template>
+    <form class="tag-selection-form" @submit="prevent">
```

Comment:
I'm not sure what you do with the @submit can you document. Do you prevent the default action of the submit or have you forgot the method?

Reply:
I wanted to avoid the return enter to reload the page, it was a mistake.